src/server/pegasus_server_impl.cpp

```diff
@@ -1375,7 +1414,12 @@ ::dsn::error_code pegasus_server_impl::start(int argc, char **argv)
         _meta_store->set_data_version(PEGASUS_DATA_VERSION_MAX);
         _meta_store->set_last_flushed_decree(0);
         _meta_store->set_last_manual_compact_finish_time(0);
+        _meta_store->set_usage_scenario(ROCKSDB_ENV_USAGE_SCENARIO_NORMAL);
```

Comment:
We can specify envs when create new table, so actual usage scenario may be included in envs which is passed from meta server, see https://github.com/apache/incubator-pegasus/blob/dea9756663847acb326539f61b80c6c48f92456e/src/server/pegasus_server_impl.cpp#L1256

You can read it from envs.